### PR TITLE
Skip records with no data on import

### PR DESCRIPTION
**Pull request by no-reply**
Our sample BePress exports contain rows with only delimiters. Rather than fail
validation on these rows, we prefer to skip them entirely. We do this at the
`Parser` layer, so anything consuming `Parser#records` (validators, importers,
etc...) will treat these rows as non-records.

### CODE REVIEW

File: app/importers/mahonia_csv_parser.rb [hunk 1]
```diff
@@ -7,7 +7,11 @@ def records
     file.rewind
     # use the MahoniaMapper
     CSV.parse(file.read, headers: true).each do |row|
+      next if row.to_h.values.all?(&:nil?)
```
Comment (jenlindner) [inline app/importers/mahonia_csv_parser.rb:10]: 💯

**Review by jenlindner — DISMISSED** [review]
Is the commit message what you want it to be?

**Comment by no-reply** [comment]
> Is the commit message what you want it to be?

Something happened with the GitHub form, I fixed the PR message. I think the commit message is correct.

**Comment by no-reply** [comment]
Sorry @jenlindner, I pushed up a spec commit which dismissed your review.

**Comment by jenlindner** [comment]
No problem, looks great!

**Review by jenlindner — APPROVED** [review]

**Pull request merged**